maven_iuvs/echelle.py

```diff
@@ -54,15 +54,15 @@ def weekly_echelle_report(weeks_before_now_to_report, root_folder):
     print(f"  covering observations after {weekly_report_idx[0]['datetime'].isoformat()[:19].replace('T',' ')} UTC")
     print(f"                              orbit {iuvs_orbno_from_fname(weekly_report_idx[0]['name'])}+\n")
 
-    latest_orbit_with_files = iuvs_orbno_from_fname(weekly_report_idx[-1])
-    print(f'Data available through ------> orbit {latest_orbit_with_files} ({iuvs_filename_to_datetime(weekly_report_idx[-1]["name"]).isoformat()[:10]})')
+    latest_orbit_with_files = iuvs_orbno_from_fname(weekly_report_idx[-1]['name'])
```

Comment:
where is this “name” element defined in your committed code? Does this code run on your machine or are there uncommitted files?

Reply:
"name" is a key in the echelle file index, it's not a separate variable or anything

Reply:
my bad, I forgot this was in the index file!!